### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by roypat**
Bring the security policy document in this repository in-sync with the one over at rust-vmm/community, e.g.
https://github.com/rust-vmm/community/blob/main/SECURITY.md

### Summary of the PR

*Please summarize here why the changes in this PR are needed.*

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR are signed (with `git commit -s`), and the commit
  message has max 60 characters for the summary and max 75 characters for each
  description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -11,12 +11,14 @@ We ask anyone who discovers a security concern associated with any of the
...
-|Maintainer	|Email	|PGP Key	|
-|---	|---	|---	|
-|Andreea Florescu |fandree@amazon.com |https://github.com/andreeaflorescu.gpg |
-|Laura Loghin |lauralg@amazon.com	|https://github.com/lauralt.gpg	|
-|Rob Bradford |robert.bradford@intel.com |https://github.com/rbradford.gpg |
-|Sebastien Boeuf |sebastien.boeuf@intel.com |https://github.com/sboeuf.gpg |
+|Maintainer|Email|PGP Key|
+|---|---|---|
+|Patrick Roy|roypat@amazon.co.uk|https://github.com/roypat.gpg|
+|Jonathan Woollett-Light|jcawl@amazon.co.uk|https://github.com/JonathanWoollett-Light.gpg|
```
Comment (andreeaflorescu) [inline SECURITY.md]: Please use alphabetical order.

Comment (JonathanWoollett-Light) [reply SECURITY.md]: I think we should add a check to explicitly enforce this, or not require it. Personally I don't see any clear benefit from it.

Comment (andreeaflorescu) [reply SECURITY.md]: It was alphabetical order before, and we've been using this across the project. I believe it's pretty standard to use alphabetical order unless there is a hierarchy or a particular order in which you need to contact people. I don't think we should have a check for something as simple as this. For example, the maintainers list in Firecracker is also alphabetical.

Comment (roypat) [reply SECURITY.md]: Fixed

**Review by andreeaflorescu — APPROVED** [review]

**Review by JonathanWoollett-Light — APPROVED** [review]

**Pull request merged**